### PR TITLE
Add "America/Nuuk" time zone

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties [hunk 1]
```diff
@@ -2236,3 +2236,4 @@
 2227 Asia/Famagusta
 2228 Europe/Saratov
 2229 Asia/Qostanay
+2230 America/Nuuk
```
Comment (findepi) [inline presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties:2239]: Sadly, this is technically not a new zone.
It's the same as `1794 America/Godthab`.

i guess we have other cases like that, do we?

Should we extend zone-index format to support multiple labels (aliases) for single id?

Comment (electrum) [reply presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties:2239]: This is a question for @dain. How would we know if others are an alias?

Comment (dain) [reply presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties:2239]: I would bet there are other aliases in that file already, but I do not know for sure.  I would just map this to a new id.  I do not believe there is a downside to having aliases with different ids as the id it not part of equality, sorting or anything.

Comment (martint) [reply presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties:2239]: Right, the is is a compact identifier for the timezone, so there's value in having each zone assigned to a unique id. It makes it possible to go back and forth if needed.

Comment (findepi) [reply presto-spi/src/main/resources/io/prestosql/spi/type/zone-index.properties:2239]: If `America/Nuuk` and `America/Godthab` are the same zone, is there a point in distinguishing them?

I think the question can be -- does `TimeZoneKey` is a _time zone identifier_ or is it a _time zone ID's identifier_.
In the first case, distinguishing the Nuuk and Godthab is unsound.
In the second case, it's warranted, but I would expect user's input "Godthab" could be replaced with "Nuuk" anyway, so we're not gaining much.

Anyway, the change seems not harmful (problem is pre existing) and also required now, so -- LGTM.

**Pull request merged**